EIPS/eip-3416.md

```diff
@@ -0,0 +1,114 @@
+---
+eip: 3416
+title: Median gas premium and miner fees
+author: HexZorro (@hexzorro), strongly based on EIP-1559 circa Aug-2020.
+discussions-to: https://ethereum-magicians.org/t/eip-3416-median-gas-premium/5755
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-18
+---
+
+## Simple Summary
+
+A transaction pricing mechanism that includes a fixed-per-block network fee that is *not*-burned plus a *median* premium fee per transaction. Also, dynamically expands/contracts block sizes to deal with transient congestion. We target the following goals:
+
+* Gas prices spikes are mathematically smoothed out.
+* Maintain gas price preference, i.e. transaction senders willing to pay extra in fees will be rewarded with early preferential inclusion in the blocks, because the miners want to maximize their profits and include transactions with higher fee caps first to maximize the median.
+* Final gas price paid by the sender is, most of the time, smaller than the maximum gas price specified by sender. 
+* Gas pricing is more robust to sender manipulation or miner manipulation.
+* All fees go to miners.
```

Comment:
Simple summary should be about a sentence long.  In terms of length/content, imagine an email subject line, GitHub pull request title, or forum topic.

Reply:
thanks